src/snapred/backend/service/ServiceFactory.py

```diff
@@ -26,14 +27,15 @@ class ServiceFactory:
 
     def __init__(self):
         # register the services
-        self.serviceDirectory.registerService(ConfigLookupService)
-        self.serviceDirectory.registerService(ReductionService)
-        self.serviceDirectory.registerService(StateIdLookupService)
-        self.serviceDirectory.registerService(CalibrationService)
-        self.serviceDirectory.registerService(CrystallographicInfoService)
-        self.serviceDirectory.registerService(CalibrantSampleService)
-        self.serviceDirectory.registerService(ApiService)
-        self.serviceDirectory.registerService(FitMultiplePeaksService)
+        self.serviceDirectory.registerService(ConfigLookupService())
+        self.serviceDirectory.registerService(ReductionService())
+        self.serviceDirectory.registerService(StateIdLookupService())
+        self.serviceDirectory.registerService(CalibrationService())
+        self.serviceDirectory.registerService(CrystallographicInfoService())
+        self.serviceDirectory.registerService(CalibrantSampleService())
+        self.serviceDirectory.registerService(ApiService())
+        self.serviceDirectory.registerService(FitMultiplePeaksService())
```

Comment:
These shouldnt be instantiated.  Pete changed it to lazily initialize.

Reply:
Okay, fixed it.